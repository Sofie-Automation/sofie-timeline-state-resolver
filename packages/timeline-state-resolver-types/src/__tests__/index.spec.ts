import * as TSR_types from '../index'

describe('index', () => {
	test('imports', () => {
		// Just verify the exports, to catch breaking changes in exports:
		const exports: string[] = Object.keys(TSR_types).sort((a, b) => {
			if (a > b) return 1
			if (a < b) return -1
			return 0
		})

		expect(exports).toMatchSnapshot()
	})
})
