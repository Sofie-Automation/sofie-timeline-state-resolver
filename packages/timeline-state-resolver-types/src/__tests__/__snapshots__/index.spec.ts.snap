// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index imports 1`] = `
[
  "AbstractActions",
  "ActionExecutionResultCode",
  "AtemActions",
  "AtemMediaPoolType",
  "AtemTransitionStyle",
  "BlendMode",
  "CasparCGActions",
  "ChannelFormat",
  "Chroma",
  "DeviceType",
  "Direction",
  "Ease",
  "EmberParameterType",
  "HyperdeckActions",
  "LawoDeviceMode",
  "MappingAtemType",
  "MappingCasparCGType",
  "MappingHyperdeckType",
  "MappingLawoType",
  "MappingMultiOscType",
  "MappingObsType",
  "MappingPanasonicPTZType",
  "MappingQuantelType",
  "MappingSingularLiveType",
  "MappingSisyfosType",
  "MappingSofieChefType",
  "MappingTricasterType",
  "MappingVmixType",
  "MediaSourceType",
  "MultiOSCDeviceType",
  "OBSRequest",
  "OSCDeviceType",
  "OSCValueType",
  "QuantelActions",
  "QuantelControlMode",
  "QuantelTransitionType",
  "ShotokuTransitionType",
  "SlotId",
  "SlotStatus",
  "StatusCode",
  "Timeline",
  "TimelineContentTypeAtem",
  "TimelineContentTypeCasparCg",
  "TimelineContentTypeHTTP",
  "TimelineContentTypeHTTPParamType",
  "TimelineContentTypeHyperdeck",
  "TimelineContentTypeLawo",
  "TimelineContentTypeMultiOSC",
  "TimelineContentTypeOBS",
  "TimelineContentTypeOSC",
  "TimelineContentTypePanasonicPtz",
  "TimelineContentTypePharos",
  "TimelineContentTypeShotoku",
  "TimelineContentTypeSingularLive",
  "TimelineContentTypeSisyfos",
  "TimelineContentTypeSofieChef",
  "TimelineContentTypeTcp",
  "TimelineContentTypeTriCaster",
  "TimelineContentTypeVMix",
  "TimelineContentTypeVizMSE",
  "Transition",
  "TranslationsBundleType",
  "TransportStatus",
  "VIZMSETransitionType",
  "VMixCommand",
  "VMixInputType",
  "VMixTransitionType",
  "VideoFormat",
  "VizMSEActions",
  "VmixActions",
]
`;
